app/routes.php

```diff
@@ -392,12 +392,13 @@ public function handle( FunFunFactory $ffFactory, Application $app, Request $req
 					// TODO: check txn_type
 					// TODO: update donation's status and payment provider related fields
 
-					$useCase = $ffFactory->newHandlePayPalPaymentNotificationUseCase();
-					return new Response( $useCase->handleNotification(
+					$useCase = $ffFactory->newHandlePayPalPaymentNotificationUseCase( $this->getValueFromCustomVars( $post->get( 'custom', '' ), 'utoken' ) );
```

Comment:
Looking at the indent here I'm thinking it's probably good to move this stuff into its own file